bin/create-docs-pr.js

```diff
@@ -0,0 +1,303 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { program } = require('commander')
+
+const Github = require('./github')
+const git = require('./git-commands')
+
+const DEFAULT_FILE_NAME = 'NEWS.md'
```

Comment:
A thought that if we eventually trigger off of say a successful release publish, we could also just grab from that release in GitHub. Then wouldn't have to parse down the text / duplicate the work that we do to create the GitHub release in the first place. May or may not be the way to go but food for thought.

Reply:
true. i just did this because we already had helpers to parse the file